liveviz/src/components/trackermap/ui/TrackerUI.js

```diff
@@ -75,11 +75,15 @@ class TrackerUI extends Component {
   }
   componentDidMount() {
     const {width, height} = this.domElement.getBoundingClientRect();
+
     this.setState(merge(this.state, {width, height}));
   }
   render() {
-    const {width, height} = this.state,
-    [arenaWidth, arenaHeight] = [20, 10]
+    const {width, height} = this.state;
+    const arenaWidth: this.props.room.Data[0][0]
```

Comment:
have you tested it? 
`const arenaWidth: this.props.room.Data[0][0]` is not js valid syntax!!!

Try something like:

`const {arenaWidth, arenaHeight} =  this.props.room.Data[0]`